README.md

```diff
@@ -254,6 +254,30 @@ reactor.listenTCP(8000, factory)
 reactor.run()
 ```
 
+#### Flask
+
+To use prometheus with [flask](http://flask.pocoo.org/) is as simple as exposing an endpoint which returns a response containing the metrics.
+
+```python
+from flask import Flask, Response, request
+from prometheus_client import generate_latest, CONTENT_TYPE_LATEST, core
+
+app = Flask(__name__)
+
+@app.route("/metrics")
+def metrics():
+    r = core.REGISTRY
```

Comment:
I don't want this logic duplicated here, I want you to use one of the existing functions for it. This code will change over time, and copy&pasted code in user applications won't get updated.

Reply:
Do you mean using like so?

```python
from prometheus_client import REGISTRY
def metrics():
    r = REGISTRY
```

If not, I'm probably missing the point. I my initial implementation I was hoping/expecting to find a clean function that returns a response having parsed out the parameters and done its magic. A new public api could be added that does something like this

```python
def generate_latest(registry, query_string)
    """Generate the latest metrics taking the query string into account"""
    pass
```

Reply:
No, I mean reusing something like MetricsHandler that does all the http stuff.

Reply:
Since flask's response is a uwsgi application, I'll try to see how I can reuse ```make_wsgi_app``` as is.